docs/userguide/modules/creating-service-bindings/pages/binding-options.adoc

```diff
@@ -236,5 +239,12 @@ spec:
 If a service binding is declared using these label selectors to pick up workloads, SBO will
 periodically attempt to find and bind new workloads that match the given label selector.
 
-As a note: it is currently forbidden to attempt a binding with both `name:` and `labelSelector`
-defined (or `selector` in the spec API).  Any attempt to do so will result in a error.
+[IMPORTANT]
+====
+Currently, it is forbidden to attempt a binding with both of the following fields defined: `name:` and `labelSelector`
+
+. `name:` and `labelSelector` in the `binding.operators.coreos.com/v1alpha1` API
+. `name:` and `selector` in the `servicebinding.io` API (Spec API)
+
+Any attempt to do so will result in a error.
```

Comment:
```suggestion
Currently, it is forbidden to attempt a binding with the following fields defined: 

. `name:` and `labelSelector` in the `binding.operators.coreos.com/v1alpha1` API
. `name:` and `selector` in the `servicebinding.io` API (Spec API)

Any attempt to do so will result in an error.
```
The usage of ` and` here explicitly tells you that you cannot define `name:` and `labelSelector` for `binding.operators.coreos.com/v1alpha1` API. Similarly `name:` and `selector` in the `servicebinding.io` API (Spec API) when attempting a binding.

Ensure to carefully consider my suggestions and apply them. I had given [this suggestion](https://github.com/redhat-developer/service-binding-operator/pull/1130/files#r872693629) previously as well.

Reply:
I wanted to remove all chance of ambiguity.  Maybe this?
```suggestion
Currently, it is forbidden to attempt a binding with the following fields defined:

. Both `name:` and `labelSelector` in the `binding.operators.coreos.com/v1alpha1` API
. Both `name:` and `selector` in the `servicebinding.io` API (Spec API)

Any attempt to do so will result in a error.
```

Reply:
@sadlerap Please apply this change and push it. I am unable to see our fixes and changes in the PR. Kindly check once. Thanks!

Reply:
Applied.